Update to current OI Hipster and use opensource virtualbox additions

Toasterson opened the pull request.

xen0l committed:
Merge pull request #1 from xen0l/master

Fix typo in README.md

xen0l committed:
Release 2017.04

xen0l committed:
Release OpenIndiana 2017.10

xen0l committed:
Update to OpenIndiana hipster 2018.10

Toasterson committed:
Update build and integrate KVM

Toasterson committed:
Update Box to newest Snapshot and create helper makefile

drscream commented on build.sh:
```diff
@@ -0,0 +1,31 @@
+
+BUILD_VERSION=$(date +%Y%m%d)
+
+if [[ ! -f "OI-hipster-${BUILD_VERSION}-virtualbox.box" ]]; then
```
Could we add some option to specify which version we like to build? For example if `VirtualBox` isn't used. Maybe it makes more sense to use an `Makefile` instead of an shell script?

Toasterson committed:
Update instructions and scripts to newest packer version

Update instructions and scripts to newest packer version add better Automation with scripts

Toasterson committed:
Fix bug that makefile breaks when datetime changes

Toasterson committed:
Remove test from release target as testing setup is broken and it hasn't changed in a long time

Toasterson committed:
Remove test from release target as testing setup is broken and it hasn't changed in a long time

Toasterson committed:
Adding support for new image-builder

Toasterson committed:
Adding image-builder submodule

Toasterson committed:
Merge branch 'master' of github.com:Toasterson/oi-packer

Toasterson committed:
fixing up metadata-agent and implementing workflow

Signed-off-by: Till Wegmueller <toasterson@gmail.com>

Toasterson committed:
Implement cloud images built by image-builder (#8)

* Remove test from release target as testing setup is broken and it hasn't changed in a long time

* Adding support for new image-builder

* Adding image-builder submodule

* fixing up metadata-agent and implementing workflow

Signed-off-by: Till Wegmueller <toasterson@gmail.com>

---------

Signed-off-by: Till Wegmueller <toasterson@gmail.com>

Toasterson committed:
Packer init changes
Remove not needed Vagrantgile

Signed-off-by: Till Wegmueller <toasterson@gmail.com>

Toasterson committed:
Add CI workflow

Signed-off-by: Till Wegmueller <toasterson@gmail.com>

Toasterson committed:
Fix github path

Signed-off-by: Till Wegmueller <toasterson@gmail.com>

Toasterson committed:
Switch to vagrant activated environment

Signed-off-by: Till Wegmueller <toasterson@gmail.com>

Toasterson committed:
Some cleanup and add ansible script for lxbuilder

Signed-off-by: Till Wegmueller <toasterson@gmail.com>

Toasterson committed:
Fix vagrant ssh key setup

Signed-off-by: Till Wegmueller <toasterson@gmail.com>

Toasterson committed:
Remove CI

Signed-off-by: Till Wegmueller <toasterson@gmail.com>

Toasterson committed:
Fixup vagrant ssh auth keys

Signed-off-by: Till Wegmueller <toasterson@gmail.com>

Toasterson committed:
Also clean output dir

Signed-off-by: Till Wegmueller <toasterson@gmail.com>
